validator/accounts/status.go

```diff
@@ -0,0 +1,240 @@
+package accounts
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+
+	grpc_retry "github.com/grpc-ecosystem/go-grpc-middleware/retry"
+	"github.com/pkg/errors"
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/sirupsen/logrus"
+	"go.opencensus.io/plugin/ocgrpc"
+	"go.opencensus.io/trace"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/metadata"
+)
+
+// ValidatorStatusMetadata holds all status information about a validator.
+type ValidatorStatusMetadata struct {
+	PublicKey []byte
+	Metadata  *ethpb.ValidatorStatusResponse
+}
+
+// MaxRequestLimit specifies the max concurrent grpc requests allowed
+// to fetch account statuses.
+const MaxRequestLimit = 5 // XXX: Should create flag to make parameter configurable.
```

Comment:
Remove the `XXX` comment, or open up an issue in the repo for this as a todo. We use 
`TODO(#Num)` where `Num` represents the issue number.